pkg/processor/sumologicschemaprocessor/config.go

```diff
@@ -66,6 +72,12 @@ func createDefaultConfig() component.Config {
 			SquashSingleValues: defaultNestingSquashSingleValues,
 		},
 		AggregateAttributes: defaultAggregateAttributes,
+		LogFieldsAttributes: &logFieldAttributesConfig{
+			SeverityNumberAttribute: &logFieldAttribute{defaultAddSeverityNumberAttribute, "loglevel"},
+			SeverityTextAttribute:   &logFieldAttribute{defaultAddSeverityTextAttribute, "severitytext"},
+			SpanIdAttribute:         &logFieldAttribute{defaultAddSpanIdAttribute, "spanid"},
+			TraceIdAttribute:        &logFieldAttribute{defaultAddTraceIdAttribute, "traceid"},
+		},
```

Comment:
This seems fishy (attribute names are hardcoded), this is what I want you to test